### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by wolf31415** [Sheights/StarboundSimpleVoreMod#103]
Could I put this on steam workshop? For some reason everytime i put a mod in the starbound mods folder it dosent work. Unless there are other required mods for this mod. please help

**Comment by KH007** [comment]
No. That option is right out. Sheights themself has explicitly stated that since this is a NSFW mod, it cannot go onto the Workshop.

Did you remember to download a Character Extender such as XBawks' Character Extender? Those are required for a mod that adds races, which the SSVM does.
